backend/kernelCI_app/management/commands/notifications.py

```diff
@@ -431,6 +431,22 @@ def evaluate_test_results(
     return new_issues, fixed_issues, unstable_tests
 
 
+def process_submission_options(
+    *,
+    default_recipients: list[str],
+    specific_recipients: list[str],
+    options: list[PossibleReportOptions],
+) -> list[str]:
+    recipients = default_recipients + specific_recipients
+
+    for option in options:
+        match option:
+            case "ignore_default_recipients":
```

Comment:
Can we simplify this to:
```suggestion
    if "ignore_default_recipients" in options:
```

Reply:
Done